docs/install/install-demo.md

```diff
@@ -54,17 +56,16 @@ Enable the Demo ISLE Installation to be viewed locally on personal computer brow
 
 ---
 
-## Step 4: Run Islandora Drupal site Install Script
+## Step 4: Run Islandora Drupal Site Install Script
 
-This process may take 10 - 20 minutes (_depending on system and internet speeds_)
+This process may take 10-20 minutes (_depending on system and internet speeds_)
 
-* Run the install site script on the Apache container by copying and pasting this command:
-```
-docker exec -it isle-apache-ld bash /utility-scripts/isle_drupal_build_tools/isle_islandora_installer.sh
-```
+* Run the install site script on the Apache container by copying and pasting the appropriate command:
+    * **For Mac/Ubuntu/CentOS/etc:** `docker exec -it isle-apache-ld bash -c "cd /utility-scripts/isle_drupal_build_tools && ./isle_islandora_installer.sh`
+    * **For Microsoft Windows:** `winpty docker exec -it isle-apache-ld bash -c "cd /utility-scripts/isle_drupal_build_tools && ./isle_islandora_installer.sh`
```

Comment:
Need close quote for bash command: ./isle_islandora_installer.sh"`

Reply:
fixed! thank you!